photon-client/src/components/cameras/CameraSettingsCard.vue

```diff
@@ -108,6 +109,48 @@ watchEffect(() => {
   // Reset temp settings on remote camera settings change
   resetTempSettingsStruct();
 });
+
+const showDeleteCamera = ref(false);
+const expected = computed<string>({
```

Comment:
just do `useCameraSettingsStore().currentCameraName` directly instead of using this computed.